nomad/core/master/master.py

```diff
@@ -285,10 +293,11 @@ def build_op_image(opid, pid, pickle):
         for i, op_pickle in enumerate(ops):
             # TODO: Build multiple arch images
             # TODO: Should we use a process pool instead of spawning a new process for each image?
-            tag = "lab11nomad/operators:%s_op_%d_img" % (pid, i)
-            images.append(tag)
-            p = Process(target=build_op_image, args=(i, pid, op_pickle))
-            processes.append(p)
+            for arch in MasterConfig.DEFAULT_ARCHITECTURES:
+                tag = "lab11nomad/operators:%s_op_%d_%s" % (pid, i, arch)
```

Comment:
Use a method to generate this string, maybe defined  in `node.Architectures`.

Reply:
Done.